In [9]:
import pandas as pd
from pathlib import Path
csv_path = Path.home() / "Downloads" /"Credit Card Approvals (Clean Data)"/"clean_dataset.csv"   # <-- change this
df_credit = pd.read_csv(csv_path)
df_credit.head(50)

Gender    Age    Debt  Married  BankCustomer               Industry  \
0        1  30.83   0.000        1             1            Industrials   
1        0  58.67   4.460        1             1              Materials   
2        0  24.50   0.500        1             1              Materials   
3        1  27.83   1.540        1             1            Industrials   
4        1  20.17   5.625        1             1            Industrials   
5        1  32.08   4.000        1             1  CommunicationServices   
6        1  33.17   1.040        1             1              Transport   
7        0  22.92  11.585        1             1  InformationTechnology   
8        1  54.42   0.500        0             0             Financials   
9        1  42.50   4.915        0             0            Industrials   
10       1  22.08   0.830        1             1                 Energy   
11       1  29.92   1.835        1             1                 Energy   
12       0  38.25   6.000        1             1             Financials   
13       1  48.08   6.040        1             1             Financials   
14       0  45.83  10.500        1             1              Materials   
15       1  36.67   4.415        0             0             Financials   
16       1  28.25   0.875        1             1  CommunicationServices   
17       0  23.25   5.875        1             1              Materials   
18       1  21.83   0.250        1             1            Real Estate   
19       0  19.17   8.585        1             1  InformationTechnology   
20       1  25.00  11.250        1             1                 Energy   
21       1  23.25   1.000        1             1                 Energy   
22       0  47.75   8.000        1             1                 Energy   
23       0  27.42  14.500        1             1              Utilities   
24       0  41.17   6.500        1             1              Materials   
25       0  15.83   0.585        1             1                 Energy   
26       0  47.00  13.000        1             1  ConsumerDiscretionary   
27       1  56.58  18.500        1             1            Real Estate   
28       1  57.42   8.500        1             1              Education   
29       1  42.08   1.040        1             1            Industrials   
30       1  29.25  14.790        1             1        ConsumerStaples   
31       1  42.00   9.790        1             1              Utilities   
32       1  49.50   7.585        1             1  ConsumerDiscretionary   
33       0  36.75   5.125        1             1              Education   
34       0  22.58  10.750        1             1              Materials   
35       1  27.83   1.500        1             1            Industrials   
36       1  27.25   1.585        1             1  InformationTechnology   
37       0  23.00  11.750        1             1              Utilities   
38       1  27.75   0.585        0             0  InformationTechnology   
39       1  54.58   9.415        1             1             Healthcare   
40       1  34.17   9.170        1             1                 Energy   
41       1  28.92  15.000        1             1                 Energy   
42       1  29.67   1.415        1             1            Industrials   
43       1  39.58  13.915        1             1            Industrials   
44       1  56.42  28.000        0             0                 Energy   
45       1  54.33   6.750        1             1                 Energy   
46       0  41.00   2.040        0             0              Materials   
47       1  31.92   4.460        1             1  InformationTechnology   
48       1  41.50   1.540        1             1  ConsumerDiscretionary   
49       1  23.92   0.665        1             1                 Energy   

   Ethnicity  YearsEmployed  PriorDefault  Employed  CreditScore  \
0      White          1.250             1         1            1   
1      Black          3.040             1 

In [5]:
rows,cols = df_credit.shape
print("Rows:", rows, "Columns:", cols)

Rows: 690 Columns: 16


In [6]:
df_credit.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 690 entries, 0 to 689
Data columns (total 16 columns):
 #   Column          Non-Null Count  Dtype  
---  ------          --------------  -----  
 0   Gender          690 non-null    int64  
 1   Age             690 non-null    float64
 2   Debt            690 non-null    float64
 3   Married         690 non-null    int64  
 4   BankCustomer    690 non-null    int64  
 5   Industry        690 non-null    object 
 6   Ethnicity       690 non-null    object 
 7   YearsEmployed   690 non-null    float64
 8   PriorDefault    690 non-null    int64  
 9   Employed        690 non-null    int64  
 10  CreditScore     690 non-null    int64  
 11  DriversLicense  690 non-null    int64  
 12  Citizen         690 non-null    object 
 13  ZipCode         690 non-null    int64  
 14  Income          690 non-null    int64  
 15  Approved        690 non-null    int64  
dtypes: float64(3), int64(10), object(3)
memory usage: 86.4+ KB


Dataset: "Credit Card Approvals"
1. This dataset contains 690 rows and 16 columns.
2. Columns include age,gender,employment status, debt status,credit score and income.
3. This dataset helps to analyze trends and factors influencing credit card approvals.

In [7]:
df_credit.columns

Index(['Gender', 'Age', 'Debt', 'Married', 'BankCustomer', 'Industry',
       'Ethnicity', 'YearsEmployed', 'PriorDefault', 'Employed', 'CreditScore',
       'DriversLicense', 'Citizen', 'ZipCode', 'Income', 'Approved'],
      dtype='object')

Columns: 
'Gender'           - qualitative    - Category 0 and 1, even it looks like numerical value -> not measurable value (Male/Female)
'Age'              - quantitative   - Represented by a numerical value that can be ordered.
'Debt'             - quantitative   - Represents the amount of debt a person has. It’s a numeric measure 
'BankCustomer'     - qualitative    - Category 0 and 1 either person is a customer of the Bank (1) or not (0).-> not measurable 
'Industry'         - qualitative    - Different types of job industries as text labels -> not mesurable numeric values
'Ethnicity'        - qualitative    - Represents groups. There categories with no numeric meaning.
'YearsEmployed'    - quantitative   - Amount of years person is employed (numeric value)
'PriorDefault'     - qualitative    - Indicates whether a person has defaulted before (1/0) - yes or no.Categorical label.
'Employed'         - qualitative    - Indicated whether the person is employed or not (1/0) -yes or no. Categorical label.
'CreditScore'      - quantitative   - A numeric measure of creditworthiness, numeric value and can be measured
'DriversLicense'   - qualitative    - Indicates whether a person has driving license (1/0) - yes or no.Categorical label.
'Citizen'          - qualitative    - Represents citizenship status as text labels
'ZipCode'          - qualitative    - Represents georgraphic location. Even tough it's stored as numbers it can't be mesaured
'Income'           - quantitative   - Represented in  a numeric value(measurable) and shows the income of a person
'Approved'         - qualitative    - Indicated if the credit card was approved or not as a category 0 or 1 (yes or no)

